repos/cli.proj

```diff
@@ -118,37 +91,5 @@
                       Overwrite="true" />
   </Target>
```

Comment:
Do you mind adding a link to https://github.com/dotnet/cli/issues/9457 tracking adding a feature that lets us remove this target while you're cleaning up?

Reply:
Or https://github.com/dotnet/source-build/issues/561 actually.

Reply:
I added a comment with this link.